client/tests/integration/triggers/time_trigger.rs

```diff
@@ -115,7 +117,7 @@ fn change_asset_metadata_after_1_sec() -> Result<()> {
         &mut test_client,
         &account_id,
         Duration::from_secs(1),
-        usize::try_from(PERIOD.as_millis() / DEFAULT_CONSENSUS_ESTIMATION.as_millis() + 1)?,
+        2, // One block for trigger registration and one for execution
```

Comment:
What if we introduce something like scopes to show why it's equal to 2? And have a variable that we increment?
I see that it's a test but that value might cause confusions even with the commentary

Reply:
No longer relevant.